docs/Projects/TAD/reporting_standard.md

```diff
@@ -0,0 +1,312 @@
+                This field is not always present.
+                2. `feature` (REQUIRED, string). The feature the graph corresponds to. This is required, since all
+                relevant graphs are dependend on features.
+                3. `data` (list)
+                    1. `x_value` (REQUIRED, float). The $x$-value of the graph.
+                    2. `y_value` (REQUIRED, float). The $y$-value of the graph.
+
+## Example
+
+### System Card
+
+```yaml
+upl: {upl_uri}                                          # Optional. Example: https://standaarden.overheid.nl/owms/terms/AangifteVertrekBuitenland
+owners:
+- organization: {organization_name}                     # Required. Example: BZK
+  oin: {oin}                                            # Optional. Example: 00000001003214345000
+  name: {owner_name}                                    # Optional. Example: John Doe
+  email: {owner_email}                                  # Optional. Example: johndoe@email.com
+  role: {owner_role}                                    # Optional. Example: Data Scientist.
+
+# Model Cards
+models:
+ - !include {model_card_uri}                            # Optional. Example: model_card.yaml.
+
+# Assessments like IAMA.
+assessments:
+- name: {assessment_name}                               # Required. Example: IAMA.
+  date: {assessment_date}                               # Required. Example: 25-03-2025.
+  contents:
+    - question: {question_text}                         # Required. Example: "Question 1: ...".
+      answer: {answer_text}                             # Required. Example: "Answer: ...".
+      remarks: {remarks_text}                           # Optional. Example: "Remarks: ...".
+      authors:                                          # Optional. Example: "['John', 'Peter']".
+        - name: {author_name}
+      timestamp: {timestamp}                            # Optional. Example: 1711630721.
+```
+
+### Model Card
+
+```yaml
+algorithm-registers:
+  - {id}                                                # Optional. Example: https://algoritmes.overheid.nl/nl/algoritme/45991543.
+language:
+  - {lang_0}                                            # Optional. Example nl.
+license: {licence}                                      # Required. Example: Apache-2.0 or any license SPDX ID from https://opensource.org/license or "other".
+license_name: {licence_name}                            # Required. if license = other, specify an id for the licence. Example: 'my-license-1.0'
+license_link: {license_link}                            # Required. if license = other, specify "LICENSE" or "LICENSE.md" to link to a file of that name inside the repo, or a URL to a remote file.
+library_name: {library_name}                            # Optional. Example: keras or any library from https://github.com/huggingface/huggingface.js/blob/main/packages/tasks/src/model-libraries.ts.
```

Comment:
Shall we apply same "setup" for license and library? Here it would mean we should rename `library_name` -> `library` where the `library` can contain a value from the mentioned source (similar to what we do with license).

And `library` will be renamed to `library_name`.

Reply:
This is what I meant yesterday that the naming can become awkward by adhering to the original names of HF. I also would like to have consistent naming, but then we also lose the compatibility with HF for the library field. Let's also discuss this in the meeting you proposed.

Reply:
We decided to remove the library reference.